gradle/libs.versions.toml

```diff
@@ -1,24 +1,22 @@
 [versions]
 agp = "8.2.2"
-kotlin = "2.0.10"
-compose = "1.6.11"
-kotlinx-serialization-json = "1.6.3"
-kotlinx-datetime = "0.6.0"
+kotlin = "2.0.21"
+compose = "1.7.0"
+kotlinx-serialization-json = "1.7.1"
+kotlinx-datetime = "0.6.1"
 nexus-publish = "2.0.0"
 documentfile = "1.0.1"
 jna = "5.14.0"
-coil = "3.0.0-alpha07"
+coil = "3.0.0-alpha10"
```

Comment:
Coil `3.0.1` is now in [stable channel](https://github.com/coil-kt/coil/blob/main/CHANGELOG.md#301---november-7-2024)

Reply:
Yes I know, but it's just for the sample so I can keep it like that for now.